biosys/apps/main/api/views.py

```diff
@@ -212,9 +213,68 @@ def perform_update(self, serializer):
 
 
 class FormViewSet(viewsets.ReadOnlyModelViewSet):
-    permission_classes = (IsAuthenticated, DRYPermissions)
+    permission_classes = (IsAuthenticated,)
     serializer_class = serializers.FormSerializer
     queryset = models.Form.objects.all().distinct()
+    filter_fields = ('id', 'name', 'dataset', 'dataset__project', 'dataset__name',)
+
+
+@api_view()
```

Comment:
man, have you had a look at the other views? They are all classed based views.
Can you refactor this function-based view into a classed-based one please. Should be easy, only extends APIView.